TicTacToe.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+
+class TicTacToe {
+    public:
+        TicTacToe() {
+            for (int i = 0; i < 9; i++) {
+                arr_[i / 3][i % 3] = '1' + i;
+            }
+        }
+        void CreateBoard() {
```

Comment:
Looks like this function is actually printing out the board rather than creating them. Maybe look into simplifying the constructor and moving some of the logic from the constructor into here.